packages/patternfly-4/react-core/src/components/ChipGroup/__snapshots__/Chip.test.js.snap

```diff
@@ -3,19 +3,25 @@
 exports[`Chip closable 1`] = `
 <GenerateId
   prefix="pf-random-id-"
-/>
+>
+  <Component />
+</GenerateId>
```

Comment:
This is a pre-existing problem, but we should probably revisit these snapshot tests later. A snapshot of a shallow render of something wrapped by `GenerateId` or `ContextConsumer` isn't accomplishing anything.